File: Assignment14/Question3.c
# include<stdio.h>

void Pattern(int iRow , int iCol)
{
    int i = 0;
    for(i = 1; i<=iRow ; i++)
    {    int j = 0;
        for(j = 0 ; j < iCol ; j++)
        {
            if((i % 2)== 1)
            {
                printf("%c\t" , 'a' + j);   
            }
            else
            {
                printf("%d\t",(j + 1));
            }
        }
    printf("\n");
    }
}
int main()
{
    int iValue1 = 0;
    int iValue2 = 0;

    printf("Enter the number of rows : ");
    scanf("%d",&iValue1);

    printf("Enter the number of Coloumns : ");
    scanf("%d",&iValue2);

    Pattern(iValue1,iValue2);
    
    return 0;    
}

File: Assignment50/Question3.java
import java.util.*;

class Pattern
{
    public void Pattern(int iRow , int iCol)
    {
        int i = 0 , j = 0 ;
        
        for(i = 1; i <= iRow; i++)
        {
            
            for(j = 1; j <= iCol; j++)
            {
                if(i == 1 || i == iRow || j == 1 || j == iCol || (i + j) == (iRow + 1))
                {
                    System.out.print("*   ");
                }
                else
                {
                    System.out.print("    "); 
                }
            }
            System.out.println(); 
        }
    }

}
class Question3
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("enter number of rows : ");
        int iValue1 = sobj.nextInt();

        
        System.out.println("enter number of coloumns : ");
        int iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();
        pobj.Pattern(iValue1,iValue2);
    }
}
